flake.nix

```diff
@@ -562,11 +565,14 @@
               (if lib.versionAtLeast lib.version "20.03pre"
               then nlohmann_json
               else nlohmann_json.override { multipleHeaders = true; })
+              prometheus-cpp
             ];
 
           checkInputs = [
             cacert
-            foreman
+            # FIXME: foreman is broken on all nix/nixpkgs pin, up to and
+            # including 2.7.0
```

Comment:
Ouch, does this mean foreman is broken today on hydra master? :/. Is there a nixpkgs issue about it?

Reply:
Yes it does, and not that I know of. However, it is fixed in Nixpkgs master -- but we're pinned to the Nixpkgs that Nix 2.6.0 is using (I tried 2.6.1 and 2.7.0, and their Nixpkgs pins have the same problem).